### ISSUE TITLE
Update Readme!

### ISSUE THREAD

**Issue by AFMW** [liaoyuan-io/react-native-call-detector#1]
Can you give us an example of how implement it? 
it works for IOS and Android?, How's the installation for each one?

**Comment by cnjsstong** [comment]
Yes it does work for Android.

The installation is just like other React Native modules. Please refer to React Native doc: https://facebook.github.io/react-native/docs/linking-libraries-ios.html .

You may need to try manual linking.

**Comment by AFMW** [comment]
I already installed it and it aint do anything, I pasted the following code and it doesnt shows anything.
I pasted like this because the one that is writen on the Readme file it shows an error :
componentWillMount() {
        this.callDetector = new CallDetector('call detector name', (event) => {
            {
                console.log(event)
            }[ event ] ();
        });
    }

    componentWillUnmount() {
        this.callDetector && this.callDetector.dispose();
    }



my RN version is : "react-native": "0.48.2",

**Comment by cnjsstong** [comment]
Your code here is not correct in syntax. Have you tried to just print out `event`?

**Comment by AFMW** [comment]
Yeah, i've tried but it ain't shows something, is like it doesnt enter to the function :(
